src/phel/core.phel

```diff
@@ -258,7 +258,7 @@ This is PHP equivalent to `$args[0] . $args[1] . $args[2] ...`."
 # Basic sequence operation
 # ------------------------
 
-(defn- equals1 [a b]
+(defn equals1 [a b]
```

Comment:
I would suggest keeping `equals1` private, and try finding another workaround/fix somewhere else 🤔 
As @jenshaase mentioned: _the real problem here is the macro expansion. The expansion happens in the wrong namespace context._

Reply:
I am stuck on this one. I saw that Clojure also does not allow private functions in macros because they end up in different namespaces.

Another solution might be to use the body of equals1 as an anonymous function in the case macro.